src/couch/rebar.config.script

```diff
@@ -155,8 +155,13 @@ end.
         };
     {unix, _} when SMVsn == "91" ->
         {
-            "-DXP_UNIX -I/usr/include/mozjs-91 -I/usr/local/include/mozjs-91 -I/opt/homebrew/include/mozjs-91/ -std=c++17 -Wno-invalid-offsetof",
-            "-L/usr/local/lib -L /opt/homebrew/lib/ -std=c++17 -lmozjs-91 -lm"
+            "$CFLAGS -DXP_UNIX -I/usr/include/mozjs-91 -I/usr/local/include/mozjs-91 -I/opt/homebrew/include/mozjs-91/ -std=c++17 -Wno-invalid-offsetof",
```

Comment:
Two questions:

1. Inheriting existing flags makes sense. Is there a good reason to only start doing this with SM91? Alternatively we could make this change for all versions by doing the inheritance in the construction of the `CouchJSEnv` term below.
2. This copies $CFLAGS into $CXXFLAGS, would it be better to inherit $CXXFLAGS?

Reply:
(1) My thinking was that the standard build environment variables inheritance could be in parity between the OSes on 91 since linux meta distributions and builders often support setting up includes/libraries/hardening flags/etc in them. But with windows I think the %INCLUDE% variable is bypassing eports setup entirely in place of needing $CFLAGS inheritance. The separation of 91 vs previous versions was that anyone who setup an earlier build may have done so while ignoring bad CFLAGS since they had no consequences while any completely new setup is hopefully (ideally) able to use the only mozilla maintained version.

(2) My main concern for inheriting build system variables is that the I think any alternatives to "-std=c++17" are possibly harmful and I think (hope) most systems are supposed to be adding up those settings with CFLAGS to make CXXFLAGS.

Reply:
OK got it, thanks for the explanation.